pkg/oauth/api/v1/types.go

```diff
@@ -74,6 +74,10 @@ type OAuthClient struct {
 	// Secret is the unique secret associated with a client
 	Secret string `json:"secret,omitempty"`
 
+	// AdditionalSecrets holds other secrets that may be used to identify the client.  This is useful for rotation
+	// and for service account token validation
+	AdditionalSecrets []string `json:"additionalSecrets,omitempty"`
```

Comment:
@openshift/api-review `AdditionalSecrets` allows us to rotate secrets and allows us to accept multiple SA tokens as valid client secrets.